content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -744,17 +744,35 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-This admission controller will deny any Pod that attempts to set certain escalating
-[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)
-fields, as shown in the
-[Configure a Security Context for a Pod or Container](/docs/tasks/configure-pod-container/security-context/)
-task.
-If you don't use [Pod Security admission](/docs/concepts/security/pod-security-admission/),
-[PodSecurityPolicies](/docs/concepts/security/pod-security-policy/), nor any external enforcement mechanism,
-then you could use this admission controller to restrict the set of values a security context can take.
-
-See [Pod Security Standards](/docs/concepts/security/pod-security-standards/) for more context on restricting
-pod privileges.
+This admission controller will deny any Pod that attempts to set the following
+[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version">}}/#securitycontext-v1-core)
```

Comment:
I would link to `/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context`

Reply:
Ok, why not! I should write some feedback because I think this new presentation is really worse than the old one :/